reverse_linkedlist/jade_solution.py

```diff
@@ -0,0 +1,40 @@
+def get_data_list(data):
+  data_list = []
+  data_list.append(data['data'])
+  next_node = data['next']
+  while inner_dict is not None:    
+      node_data = next_node['data']
+      data_list.append(node_data)
+      next_node = next_node['next']
+  return data_list
+
+def make_dict(current_data, prev_dict):
+    return {
+        'data':current_data,
+        'next': prev_dict
+}
+
+def reverse_linked_list(data):
+  data_list = get_data_list(data)
+  for i,e in enumerate(data_list):
```

Comment:
Could you name these something clearer than `i` and `e`?

Reply:
okay, fine.
next time.